android/app/src/main/AndroidManifest.xml

```diff
@@ -21,13 +21,8 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
     <!-- these permissions should be removed -->
-    <!-- react-native-orientation adds an unnecessary permission; here we remove it -->
-    <uses-permission android:name="android.permission.BLUETOOTH" tools:node="remove"/>
```

Comment:
Are those not necessary anymore?

Reply:
Yes, looks like BLUETOOTH, WRITE_CONTACTS and RECORD_VIDEO were introduced by upstream projects sometime in the past, but this problem has now been fixed.
```
~/dev/status-react/node_modules $ grep -r BLUETOOTH  * | grep AndroidManifest
~/dev/status-react/node_modules $ grep -r RECORD_VIDEO  * | grep AndroidManifest
~/dev/status-react/node_modules $ grep -r WRITE_CONTACTS  * | grep AndroidManifest
react-native-contacts/README.md:Add permissions to your `android/app/src/main/AndroidManifest.xml` file.  Add only the permissions you need (i.e. if you don't need the _WRITE_CONTACTS_ permission then there's no need to add it).
~/dev/status-react/node_modules $ 
```
 So there is no longer a need to remove them from AndroidManifest.xml, and trying to do so gives the warning.